tests/lib/__snapshots__/scope-analysis.js.snap

```diff
@@ -4480,6 +4480,101 @@ Object {
 }
 `;
 
+exports[`TypeScript scope analysis tests/fixtures/scope-analysis/import-equals.ts 1`] = `
+Object {
+  "$id": 2,
+  "block": Object {
+    "range": Array [
+      0,
+      28,
+    ],
+    "type": "Program",
+  },
+  "childScopes": Array [
+    Object {
+      "$id": 1,
+      "block": Object {
+        "range": Array [
+          0,
+          28,
+        ],
+        "type": "Program",
+      },
+      "childScopes": Array [],
+      "functionExpressionScope": false,
+      "isStrict": true,
+      "references": Array [],
+      "throughReferences": Array [],
+      "type": "module",
+      "upperScope": Object {
+        "$ref": 2,
+      },
+      "variableMap": Object {
+        "foo": Object {
+          "$ref": 0,
+        },
+      },
+      "variableScope": Object {
+        "$ref": 1,
+      },
+      "variables": Array [
+        Object {
+          "$id": 0,
+          "defs": Array [
+            Object {
+              "name": Object {
+                "name": "foo",
+                "range": Array [
+                  7,
+                  10,
+                ],
+                "type": "Identifier",
+              },
+              "node": Object {
+                "range": Array [
+                  0,
+                  27,
+                ],
+                "type": "TSImportEqualsDeclaration",
+              },
+              "parent": null,
+              "type": "ImportBinding",
+            },
+          ],
+          "eslintUsed": undefined,
+          "identifiers": Array [
+            Object {
+              "name": "foo",
+              "range": Array [
+                7,
+                10,
+              ],
+              "type": "Identifier",
+            },
+          ],
+          "name": "foo",
+          "references": Array [],
```

Comment:
In TS, isn't the `require` a reference?

Reply:
No — `import foo = require('bar')` is different from `const foo = require('bar')

Reply:
i will investigate that

Reply:
@mysticatea  `import foo = require('bar')` is different syntax to write `import foo from 'bar'`
and `require` is not reference like `from` is not.

typescript have this weird syntax to distinct `export =` from `export default`

Reply:
Thank you for the explanation! I got it.